src/components/SnackBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { Alert, Snackbar } from "@mui/material";
+import { useSnack } from "../contexts/SnackContext";
+
+const SnackBarComponent = () => {
+  const { snack, handleClose } = useSnack();
+
+  return (
+    <div>
+      {snack.map((s) => {
+        return (
+          <Snackbar
+            key={snack.indexOf(s)}
+            open={s.open}
+            autoHideDuration={6000}
+            onClose={handleClose}
+          >
+            <Alert
+              onClose={handleClose}
+              severity={s.color}
+              sx={{ width: "100%" }}
+            >
+              {s.message}
+            </Alert>
+          </Snackbar>
+        );
+      })}
+    </div>
```

Comment:
```suggestion
    <Snackbar
      open={snack.open}
      autoHideDuration={3000}
      onClose={(e, r) => handleClose(r)}
    >
      <Alert
        severity={snack.color}
        sx={{ width: "100%" }}
        onClose={() => handleClose()}
      >
        {snack.message}
      </Alert>
    </Snackbar>
```

Om vi ska bara visa upp 1 alert igen